@here/harp-geoutils/lib/math/Vector3Like.ts

```diff
@@ -4,26 +4,18 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { isVector2Like, Vector2Like } from "./Vector2Like";
+
 /**
  * Interface representing a Vector3.
  */
-export interface Vector3Like {
-    /**
-     * The X position.
-     */
-    x: number;
-
-    /**
-     * The Y position.
-     */
-    y: number;
-
+export interface Vector3Like extends Vector2Like {
```

Comment:
there shouldn't be any reason for this. Any Vector3Like is already a Vector2Like

Reply:
done